moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -1029,6 +1039,7 @@ void PlanningScene::decoupleParent()
   if (!acm_)
     acm_.reset(new collision_detection::AllowedCollisionMatrix(parent_->getAllowedCollisionMatrix()));
 
+  parent_.reset();
```

Comment:
Please remove. This causes a segfault when `parent_` is accessed a couple lines below and `reset()` is called at the end of this function anyway.